wifiphisher/common/phishinghttp.py

```diff
@@ -15,59 +16,63 @@
 
 class DowngradeToHTTP(tornado.web.RequestHandler):
 
-    def get(self, url):
+    def get(self):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
 
-    def get(self, url):
+    def get(self):
+        client_request = self.request.path[1:]
         try:
-            if "/" in url:
-                self.redirect("/index")
-            self.render("index.html", **template.get_context())
+            if self.request.path == "/":
+                if os.path.exists(template.get_path() + "index.html"):
+                    self.render("index.html", **template.get_context())
```

Comment:
Also instead of calling `self.render` so many times you can have a variable like `file_name` where you store the file name in this case `index.html` and call `self.render` only once at the bottom.